views/007_events.sql

```diff
@@ -218,48 +218,6 @@ WHERE
 GROUP BY
   name;
 
--- Insert team updates in event_queue
-CREATE
-OR REPLACE FUNCTION insert_team_in_event_queue () RETURNS TRIGGER AS $$
-BEGIN
-  IF TG_OP = 'DELETE' THEN
-    INSERT INTO event_queue(name, properties) VALUES ('team.delete', jsonb_build_object('team_id', OLD.id))
-    ON CONFLICT (name, properties) DO UPDATE SET created_at = NOW(), last_attempt = NULL, attempts = 0;
-  ELSE
-    INSERT INTO event_queue(name, properties) VALUES ('team.update', jsonb_build_object('team_id', NEW.id))
-    ON CONFLICT (name, properties) DO UPDATE SET created_at = NOW(), last_attempt = NULL, attempts = 0;
-  END IF;
-  
-  RETURN NULL;
-END
-$$ LANGUAGE plpgsql;
-
-CREATE
-OR REPLACE TRIGGER team_enqueue
-AFTER INSERT OR UPDATE OR DELETE ON teams FOR EACH ROW
-EXECUTE PROCEDURE insert_team_in_event_queue ();
-
--- Create new event on any updates on the notifications table
-CREATE OR REPLACE FUNCTION notifications_trigger_function()
-RETURNS TRIGGER AS $$
-BEGIN
-    IF TG_OP = 'DELETE' THEN
-        INSERT INTO event_queue(name, properties) VALUES ('notification.delete', jsonb_build_object('id', OLD.id))
-        ON CONFLICT (name, properties) DO UPDATE SET created_at = NOW(), last_attempt = NULL, attempts = 0;
-    ELSE
-        INSERT INTO event_queue(name, properties) VALUES ('notification.update', jsonb_build_object('id', NEW.id))
-        ON CONFLICT (name, properties) DO UPDATE SET created_at = NOW(), last_attempt = NULL, attempts = 0;
-    END IF;
-
-    RETURN NULL;
-END
-$$ LANGUAGE plpgsql;
-
-CREATE OR REPLACE TRIGGER notification_update_enqueue
-AFTER INSERT OR UPDATE OR DELETE ON notifications
-FOR EACH ROW
-EXECUTE PROCEDURE notifications_trigger_function ();
-
```

Comment:
Removed this event for the same reason. The consumer was simply clearing notification cache.
It's now moved to PG Notify listener.